joint_state_publisher_gui/scripts/joint_state_publisher_gui

```diff
@@ -47,8 +47,9 @@ if __name__ == '__main__':
     try:
         rospy.init_node('joint_state_publisher_gui')
         app = QApplication(sys.argv)
+        app.setApplicationDisplayName("Joint State Publisher")
         num_rows = joint_state_publisher.get_param('num_rows', 0)
-        jsp_gui = joint_state_publisher_gui.JointStatePublisherGui('Joint State Publisher',
+        jsp_gui = joint_state_publisher_gui.JointStatePublisherGui(rospy.get_name(),
```

Comment:
While this disambiguates multiple instances of joint_state_publisher from each other, it's not totally clear (at least to me) that the name in the beginning there is the node name.  At the risk of being very verbose, I might suggest something like:

```suggestion
        jsp_gui = joint_state_publisher_gui.JointStatePublisherGui("Node: " + rospy.get_name(),
```